core/src/main/java/org/stellar/anchor/auth/AuthConfig.java

```diff
@@ -8,7 +10,9 @@
 @NoArgsConstructor
 public class AuthConfig {
   AuthType type;
-  String secret;
+  String secretString;
```

Comment:
What's the reason to change to `secretString`?

The `String` of the `String secretString` seems redudant.

Reply:
I think it's because we use `SecretKey`s now and need a way to differentiate them.